manifests/http_port.pp

```diff
@@ -51,12 +51,12 @@
   }
 
   if $facts['selinux'] == true {
-    selinux::port{"selinux port squid_port_t ${_port}":
+    ensure_resource('selinux::port', "selinux port squid_port_t ${_port}", {
```

Comment:
Hi, Can a test (rspec or acceptance)  be added to confirm that adding two `http_ports` does indeed  not cause a problem.

Reply:
Added rspec, discovered that previous rspecs for SELinux I created weren't working, fixed that too...